behandlingsprosess/src/main/java/no/nav/foreldrepenger/behandling/steg/foreslåvedtak/ForeslåVedtakTjeneste.java

```diff
@@ -93,10 +95,28 @@ class ForeslåVedtakTjeneste {
 
         håndterToTrinn(behandling, aksjonspunktDefinisjoner);
 
+        if (harTidligereOverstyringAvVedtaksbrevUtenAtDetBlirToTrinn(behandling, aksjonspunktDefinisjoner)) {
+            if (behandling.harAvbruttAksjonspunktMedType(AksjonspunktDefinisjon.FORESLÅ_VEDTAK)) {
```

Comment:
For sikkerhets skyld kan du bruke harAksjonspunktMedType - selv om de bør være avbrutt - men kan være reåpnet (opprettet) 
Her og nedenfor

Reply:
Done